src/pages/setpassword/SetPasswordForm/SetPasswordFormWide.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import {
+    Text, TextInput, View,
+} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import _ from 'underscore';
+import lodashGet from 'lodash.get';
+import styles from '../../../styles/styles';
+import ButtonWithLoader from '../../../components/ButtonWithLoader';
+import {setPassword} from '../../../libs/actions/Session';
+import ONYXKEYS from '../../../ONYXKEYS';
+import SetPasswordPageProps from '../SetPasswordPageProps';
+
+class SetPasswordFormWide extends React.Component {
```

Comment:
What is different about this form compared to the "narrow" version? Splitting them out into two components makes it difficult to see.

Reply:
I was just following the same pattern that the signin page is using, I can refactor it into a single component if you think that's best!